.github/workflows/approval-comment.yaml

```diff
@@ -12,10 +12,15 @@ jobs:
           fetch-depth: 0
       - name: Save info about the review comment as an artifact for other workflows that run on workflow_run to download them
         env:
+          # We store these values in environment variables to avoid bash script injection
+          # Specifically, it's important that we do this for github.event.review.body since this is user-controller input
+          # https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions
```

Comment:
Maybe this is worth putting in the top level of the .github directory, since this seems to be a comment applicable across many of these files in this PR.

Reply:
Added a `security-notice.md` file to the top of this directory